tools/c7n_gcp/c7n_gcp/resources/resourcemanager.py

```diff
@@ -237,11 +365,62 @@ class ProjectPropagateLabels(HierarchyAction):
     Will result in project-a being tagged with owner: network and env: dev
     and project-b being tagged with owner: ml and env: dev
 
+    Note you can also use folder ids in the form "folders/33311281122" in
+    your label json file, ie.
+
+    .. code-block:: json
+
+      {"folders/123333333": {"env": "dev", "owner": "dev"},
+       "folders/333344444": {"owner": "network"}}
+
+
+    when applying to a subtree the folder json can also be specified relative
+    to a root parent. ie. given a tree, where we only want to apply folder
+    labels to the projects under network.
+
+    ::
+
+       /dev
```

Comment:
i think this example would be easier to follow if the folder ids were used here or if the folder names where used in the policy, since it is hard to know what folder 1223123321 corresponds to

Reply:
added in some folder ids as additional attributes in the hierarchy example